app/integration_action_test.go

```diff
@@ -95,6 +95,35 @@ func TestPostAction(t *testing.T) {
 	}))
 	defer ts.Close()
 
+	setupPluginApiTest(t,
```

Comment:
I'm not following what this is unit testing -- I had expected to see a `model.PostAction` with an `Integration.URL` that read `/plugins/<plugin_id>/...`. I think all of these tests are still just pointing at `ts.URL`.

Reply:
These setupPluginApiTest don't test any directly.  The setup a plugin stub so that the call to hooks.ServerHTTP work in a loopback kind of manner.

After the previous mention to combine the relative URLs, these stubs can be removed from all integration_action_tests except for TestSubmitInteractiveDialog, and TestPostActionRelativeURL (both have relative urls and call the hook).